ibis/backends/pandas/execution/decimal.py

```diff
@@ -133,3 +134,8 @@ def execute_e(op, **kwargs):
 @execute_node.register(ops.Pi)
 def execute_pi(op, **kwargs):
     return np.pi
+
+
+@execute_node.register(ops.RandomScalar)
+def execute_random_scalar(op, **kwargs):
+    return random.random()
```

Comment:
I think we probably need to leave this out for now, since the execution results differ between the SQL backends and the non-SQL backends in a projecion:

DuckDB:

```
In [40]: con = ibis.duckdb.connect()

In [41]: t = con.read_in_memory(df)

In [42]: t
Out[42]:
┏━━━━━━━┓
┃ a     ┃
┡━━━━━━━┩
│ int64 │
├───────┤
│     1 │
│     2 │
│     3 │
└───────┘

In [43]: t.select(ibis.random())
Out[43]:
┏━━━━━━━━━━━━━━━━┓
┃ RandomScalar() ┃
┡━━━━━━━━━━━━━━━━┩
│ float64        │
├────────────────┤
│       0.783109 │
│       0.080348 │
│       0.178686 │
└────────────────┘
```

Pandas:

```
In [33]: import ibis; ibis.options.interactive = True

In [34]: import pandas as pd

In [35]: df = pd.DataFrame({"a": [1, 2, 3]})

In [36]: con = ibis.pandas.connect({'t': df})

In [37]: t = con.table('t')

In [38]: t
Out[38]:
┏━━━━━━━┓
┃ a     ┃
┡━━━━━━━┩
│ int64 │
├───────┤
│     1 │
│     2 │
│     3 │
└───────┘

In [39]: t.select(ibis.random())
Out[39]:
┏━━━━━━━━━━━━━━━━┓
┃ RandomScalar() ┃
┡━━━━━━━━━━━━━━━━┩
│ float64        │
├────────────────┤
│       0.455676 │
│       0.455676 │
│       0.455676 │
└────────────────┘
```

`random` should be evaluated once per row